lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -337,7 +337,7 @@
                                     course_id: view.context.courseID,
                                     topic_id: topicID,
                                     url: view.context.teamsUrl,
-                                    per_page: 10
+                                    perPage: 10
```

Comment:
I prefer using ``perPage`` but now this is inconsistent with ``course_id`` and ``topic_id``. If it is straightforward can you switch them all to use CamelCase.